File: app/src/main/java/com/kkkcut/e20j/ui/fragment/SupportFragment.java
package com.kkkcut.e20j.ui.fragment;

import android.widget.LinearLayout;
import butterknife.BindView;
import com.kkkcut.e20j.us.R;
import com.liying.core.MachineInfo;
import me.yokeyword.fragmentation.ISupportFragment;

/* loaded from: classes.dex */
public class SupportFragment extends BaseBackFragment {
    public static final String TAG = "SupportFragment";

    @BindView(R.id.ll_chinese)
    LinearLayout llChinese;

    @BindView(R.id.ll_foreign)
    LinearLayout llForeign;

    @BindView(R.id.ll_foreign_us)
    LinearLayout llForeignUS;

    @Override // com.kkkcut.e20j.androidquick.ui.base.QuickFragment
    protected int getContentViewLayoutID() {
        return R.layout.fragment_support;
    }

    public static ISupportFragment newInstance() {
        return new SupportFragment();
    }

    @Override // com.kkkcut.e20j.androidquick.ui.base.QuickFragment
    protected void initViewsAndEvents() {
        if (MachineInfo.isChineseMachine()) {
            this.llChinese.setVisibility(0);
            this.llForeign.setVisibility(8);
            this.llForeignUS.setVisibility(8);
        } else if (MachineInfo.isE20Us(getContext())) {
            this.llChinese.setVisibility(8);
            this.llForeign.setVisibility(8);
            this.llForeignUS.setVisibility(0);
        } else {
            this.llChinese.setVisibility(8);
            this.llForeign.setVisibility(0);
            this.llForeignUS.setVisibility(8);
        }
    }

    @Override // com.kkkcut.e20j.ui.fragment.BaseBackFragment
    public String setTitleStr() {
        return getString(R.string.support);
    }
}

File: app/src/main/java/com/kkkcut/e20j/ui/fragment/SupportFragment.kt
package com.kkkcut.e20j.ui.fragment

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import com.cutting.machine.MachineInfo
import com.kkkcut.e20j.us.R
import com.kkkcut.e20j.us.databinding.FragmentSupportBinding
import me.yokeyword.fragmentation.ISupportFragment

/* loaded from: classes.dex */
class SupportFragment() : BaseBackFragment() {
    var binding: FragmentSupportBinding? = null

    override fun onCreateView(
        layoutInflater: LayoutInflater,
        viewGroup: ViewGroup?,
        bundle: Bundle?
    ): View? {
        super.onCreateView(layoutInflater, viewGroup, bundle)
        this.binding = FragmentSupportBinding.inflate(layoutInflater, viewGroup, false)
        return binding!!.getRoot()
    }

    // com.kkkcut.e20j.androidquick.ui.base.QuickFragment
    override fun getContentViewLayoutID(): Int {
        return R.layout.fragment_support
    }

    // com.kkkcut.e20j.androidquick.ui.base.QuickFragment
    override fun initViewsAndEvents() {
        if (MachineInfo.isChineseMachine()) {
            binding!!.llChinese.setVisibility(0)
            binding!!.llForeign.setVisibility(8)
            binding!!.llForeignUs.setVisibility(8)
        } else if (MachineInfo.isE20Us(getContext())) {
            binding!!.llChinese.setVisibility(8)
            binding!!.llForeign.setVisibility(8)
            binding!!.llForeignUs.setVisibility(0)
        } else {
            binding!!.llChinese.setVisibility(8)
            binding!!.llForeign.setVisibility(0)
            binding!!.llForeignUs.setVisibility(8)
        }
    }

    // com.kkkcut.e20j.ui.fragment.BaseBackFragment
    override fun setTitleStr(): String? {
        return getString(R.string.support)
    }

    companion object {
        val TAG: String = "SupportFragment"

        fun newInstance(): ISupportFragment {
            return SupportFragment()
        }
    }
}
